cypress/e2e/formView.cy.ts

```diff
@@ -0,0 +1,40 @@
+describe('Library Details page', () => {
+    beforeEach(() => {
+        cy.intercept('GET', 'https://book-haven-be-29aa9bd8a3c7.herokuapp.com/api/v0/libraries', {
```

Comment:
Let's change all network request URLs back to the mock server endpoints until we finish testing - Please update for your test files, and I will do the same on mine.